src/libs/actions/IOU.js

```diff
@@ -961,16 +971,25 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
             lastMessageHtml: optimisticIOUReportAction.message[0].html,
         },
     };
-    const optimisticReportActionsData = {
-        onyxMethod: Onyx.METHOD.MERGE,
-        key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${optimisticIOUReport.reportID}`,
-        value: {
-            [optimisticIOUReportAction.reportActionID]: {
-                ...optimisticIOUReportAction,
-                pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+    const optimisticReportActionsData = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${optimisticIOUReport.reportID}`,
+            value: {
+                [optimisticIOUReportAction.reportActionID]: {
+                    ...optimisticIOUReportAction,
+                    pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+                },
             },
         },
-    };
+        {
```

Comment:
> It might be better to break this array into two objects. More readability.

I can do this, but I'd rather do it in a separate PR. We want to get this one merged as it is part of the wave3 initiative and we want to wrap it up.

Reply:
yes I missed the fixes, it was also not clear what you were pointing out. Good catch though! They are now fixed.